upsertGitHubTag-f79c4f36-3e61-43f4-8f6c-0b2e0e9774d7/deployment/index.js

```diff
@@ -102,21 +98,22 @@ function processEvent(event, callback) {
     console.log(JSON.stringify(event));
     var loneEvent = event.Records[0];
     var requestBody = JSON.parse(loneEvent.body);
-    if (! verifyGitHub(requestBody)) {
+
+    // The payload is encoded in base64
+    const buff = Buffer.from(requestBody.payload, 'base64');
+    const bodyDecoded = buff.toString('utf8');
+    const body = JSON.parse(bodyDecoded);
+
+    if (! verifyGitHub(requestBody, bodyDecoded)) {
```

Comment:
There is typically not a space after exclamation point, but we don't have a linter